sympy/polys/polyroots.py

```diff
@@ -381,7 +381,7 @@ def roots_binomial(f):
     base = -cancel(b/a)
     alpha = root(base, n)
 
-    if alpha.is_number:
+    if alpha.is_number and trig:
         alpha = alpha.expand(complex=True)
```

Comment:
It would be better to remove the call to `expand` altogether I think.

Reply:
I've tried doing it but some of the test cases failed 
``` FAILED sympy/polys/tests/test_polyroots.py::test_issue_8285 - assert False
FAILED sympy/polys/tests/test_polyroots.py::test_issue_8289 - assert False
FAILED sympy/polys/tests/test_polyroots.py::test_roots_quartic - assert [(-1)**(1/3),.../6)*sqrt(3)/2] == [-1, 0, 1/2 +...- sqrt(3)*I/2]
FAILED sympy/polys/tests/test_polyroots.py::test_roots_binomial - assert [-(-2)**(1/4)...4)*5**(3/4)/5] == [-10**(3/4)/1...0**(3/4)*I/10]
FAILED sympy/polys/tests/test_polyroots.py::test_issue_20913 - assert [-(-1)**(1/3).../3)*sqrt(3)/2] == [-2**(2/3)] 
```

Reply:
I've tried doing it but some of the test cases failed 
``` FAILED sympy/polys/tests/test_polyroots.py::test_issue_8285 - assert False
FAILED sympy/polys/tests/test_polyroots.py::test_issue_8289 - assert False
FAILED sympy/polys/tests/test_polyroots.py::test_roots_quartic - assert [(-1)**(1/3),.../6)*sqrt(3)/2] == [-1, 0, 1/2 +...- sqrt(3)*I/2]
FAILED sympy/polys/tests/test_polyroots.py::test_roots_binomial - assert [-(-2)**(1/4)...4)*5**(3/4)/5] == [-10**(3/4)/1...0**(3/4)*I/10]
FAILED sympy/polys/tests/test_polyroots.py::test_issue_20913 - assert [-(-1)**(1/3).../3)*sqrt(3)/2] == [-2**(2/3)] ```

Reply:
I have a more complete PR that changes this although it also changes other things gh-25196.